test/test_PokerSetup.py

```diff
@@ -0,0 +1,54 @@
+from testing_base import *
+from source.PokerPlayer4Functions import *
+from source.PokerSetup import *
+
+def test_Functions():
+
+    player1 = Player('John')
+    player2 = Player('Jacob')
+    player3 = Player('Jonathan')
+    player4 = Player('Andrew')
+    player5 = Player('Joe')
+
+    PlayerList = [player1, player2, player3, player4, player5]
```

Comment:
Just an FYI, I created that function to set all of this up and it can go in the game control. Obviously can't use it to test though because it needs user input.